snippets/ztp-example-siteconfig.adoc

```diff
@@ -43,7 +43,7 @@ spec:
         bootMode: "UEFI" <10>
         rootDeviceHints:
           wwn: "0x11111000000asd123"
-        cpuset: "0-3"  <11>
+        cpuset: "0-1,52-53"  <11>
```

Comment:
To active workload partitioning  for SNO the must much `cpuset` of PerformanceProfile with PGT and `reserved` of crio-o with SiteConfig (and make sure that the CPU cores selected are not arbitrary e.g selected NUMA cores in this case for max performance)

 Is there something like mentioned is this PR?

Reply:
We mention this explicitly here: https://github.com/openshift/openshift-docs/pull/53376/commits/5334027d7016582c7c054c3ff31369e48d316827#diff-86d1f3820339a335de3b823ffe6eac402324cc3541cebe1069ecd2dc7075f695R11

Updated slightly. PTAL